src/qibo/abstractions/circuit.py

```diff
@@ -912,3 +912,97 @@ def read_args(args):
                 gate_list[i] = ("M", qubit_list, register)
 
         return len(qubits), gate_list
+
+    def draw(self, line_wrap=None) -> str: # pragma: no cover
+        """Draw text circuit using unicode symbols.
+
+        Args:
+            line_wrap (int): maximum number of characters per line. This option
+                split the circuit text diagram in chunks of line_wrap characters.
+
+        Return:
+            String containing text circuit diagram.
+        """
+        labels = {"h": "H", "x": "X", "y": "Y", "z": "Z",
+                  "rx": "RX", "ry": "RY", "rz": "RZ",
+                  "u1": "U1", "u2": "U2", "u3": "U3",
+                  "cx": "NOT", "swap": "x", "cz": "Z",
```

Comment:
```suggestion
                  "cx": "X", "swap": "x", "cz": "Z",
```
I believe `X` is more commonly used than `NOT` when plotting quantum circuits, even for the `CNOT` gate (despite calling it `CNOT`). If we use `X` we are also more consistent with how we plot `TOFFOLI` and multi-control `X`.